modules/registry/registry.go

```diff
@@ -203,3 +253,67 @@ func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustom
 
 	return c, nil
 }
+
+// SetDockerAuthConfig sets the DOCKER_AUTH_CONFIG environment variable with
+// authentication for with the given host, username and password.
+// It returns the base64 encoded credentials and a function to reset back
+// to the previous state.
+func SetDockerAuthConfig(host, username, password string, additional ...string) (creds string, cleanup func(), err error) { //nolint:nonamedreturns // Adds context to the return values.
```

Comment:
Do we need the nolint for no-named-returns here? 
```suggestion
func SetDockerAuthConfig(host, username, password string, additional ...string) (creds string, cleanup func(), err error) { // Adds context to the return values.
```